src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
```

Comment:
the interface should be more specific, we have finite number of colors and shapes